CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to
 
 ## Unreleased
 
+- Added `disabledSources` field to the `IntegrationInstance` type from the
```

Comment:
What will be the version with these changes?

Reply:
It will be v9.1.0, I'll add that in the release-x.y.z PR